app/views/layouts/card_wide.html.slim

```diff
@@ -1,3 +1,3 @@
-- card_cls 'card-wide'
+- card_cls 'card-wide sm-px2 px0'
```

Comment:
@hursey013  So, I actually have all of that in another PR, which I closed because it was different enough from the work jessie did to be annoying. Do you think we should just add it for now to each of these rows?

I was kind of thinking that this stuff wan't going to be merged to QA before I could get in the next PR?